website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,306 @@
+
+!!!
+
+
+### Use “we” for HashiCorp
+
+Use “we” to describe recommendations by HashiCorp. Our language should never try to hedge or evade responsibility for the guidance we’re providing.
+
+!!! Do
+
+We recommend enabling SSO to...
+
+!!!
+
+!!! Dont
+
+It is a recommendation to enable SSO to...
+
+!!!
+
+### Active voice
+
+Use active voice whenever possible.
+
+!!! Do
+
+Terraform has a provider framework to leverage this behavior.
+
+We recommend configuring VCS access when first setting up an organization.
+
+Terraform has a provider framework to leverage this behavior.
+
+Next, Kubernetes will register the service.
+
+!!!
+
+!!! Dont
+
+To hook into this behavior, a provider framework has been built into Terraform.
+
+It is recommended to configure VCS access when first setting up an organization.
+
+To hook into this behavior, a provider framework has been built into Terraform.
+
+Next, the service will be registered.
+
+!!!
+
+### Use simple present tense
+
+Don't avoid the imperative if you're telling someone to do something.
+
+!!! Do
+
+Run the command `foo.`
+
+!!!
+
+!!! Dont
+
+You will be running the command `foo.`
+
+!!!
+
+### Uncommon acronyms
+
+The first time you use an acronym or abbreviation, include the abbreviated word in parentheses. Subsequent mentions can use the acronym by itself. This does not apply to common acronyms like ID, IP, or API.
+
+!!! Do
+
+This will set the Subject Alternate Name (SAN) on the certification. The SAN is used to tell clients...
+
+!!!
+
+!!! Dont
+
+This will set the SAN on the certification. The SAN is used to tell clients...
+
+!!!
+
+### Consistent terminology
+
+When labeling buttons, use the format [Verb + Noun] wherever possible. For example:  Add cluster, etc. “Add” and “Create” are frequently used. The general guidelines for their use are as follows:
+
+- **Create/Delete**: Used when a new entity is produced or destroyed in the system. Example create/delete a project (the project didn’t exist before/will no longer exist).
+- **Add/Remove**: Used when a new relationship or dependency between existing objects is produced or destroyed. Example: Add/Remove a user to/from a project (both the user and the project already existed and will continue to exist, but the relationship itself is established or dissolved).
+
+!!! Do
+
+“Create cluster” or “Delete cluster”
+
+Add user to organization
+
+!!!
+
+!!! Dont
+
+“Create cluster” on a button and “Deploy cluster” on the resulting dialog
+
+Using “remove” and “delete” or “create” and “add” interchangeably
+
+!!!
+
+For actions within an object, use the icon (when possible), the verb and the noun that will be affected.
+
+!!! Do
+
+“Edit project” or “View Project”
+
+![A menu with two actions in it with icons](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-do.png)
+
+!!!
+
+!!! Dont
+
+Miss the object that the user is doing the action. View what?
```

Comment:
[suggestion] This is somewhat informal, what do you think of this? It's a bit more wordy, but seems more clear to me.

```suggestion
Don't leave out the name of the object or element that an action applies to. In this case it is not clear what is being viewed.
```